### DISCUSSION TITLE
Save user's drawings & settings

### DISCUSSION THREAD

**Discussion by MisterOzzy** [reactivemarkets/react-financial-charts#482] [Q&A] [answered]
Is there any option to save (get in one call from chart and persist on server or wherenever) for e.g. Trend lines, indicator settings and so on which user made and then restore after refreshing?

**Comment by markmcdowell** [comment, marked as answer]
This is not provided by the library, up to you to manage and save settings.

**Reply by MisterOzzy** [reply]
How can I get from chart all drawn lines, applied indicators and so on?

**Reply by markmcdowell** [reply]
You have to manually apply all indicators etc, so your code should know what has been added.

**Reply by somguyth** [reply]
This discussion[1] be relevant in the future if someone works on this. I too would find it incredibly valuable to have this feature:

[1] https://github.com/rrag/react-stockcharts/issues/731
